tools/topology/topology2/platform/intel/tgl.conf

```diff
@@ -0,0 +1,4 @@
+# TGL-specific variable definitions
+Define {
+	SSP1_ENABLED			"true"
```

Comment:
so we are going to have to add SSP1_ENABLED for LNL as well? It'd be smarter to disable SSP1 when needed, fewer configurations to maintain in the end.

Reply:
> so we are going to have to add SSP1_ENABLED for LNL as well? It'd be smarter to disable SSP1 when needed, fewer configurations to maintain in the end.

@plbossart  @aiChaoSONG  We're not sure if LNL will have the same pin mux conflict between SSP1 and MIC as MTL, but probably it will. So it's helpful if we can disable SSP1 when needed.